### PR TITLE
Support @javadoc tag for normal service

### PR DESCRIPTION
**Pull request by tuichenchuxin**
for #683 
for https://github.com/TongchengOpenSource/smart-doc/discussions/795

### CODE REVIEW

File: src/main/java/com/ly/doc/builder/javadoc/JavadocDocBuilderTemplate.java [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+public class JavadocDocBuilderTemplate extends BaseDocBuilderTemplate {
+
+    private static final String DEPENDENCY_TITLE = "Add dependency";
+    private static final long NOW = System.currentTimeMillis();
+
+    @Override
+    public void checkAndInit(ApiConfig config, boolean checkOutPath) {
+        if (StringUtil.isEmpty(config.getFramework())) {
+            config.setFramework(FrameworkEnum.JAVADOC.getFramework());
+        }
```
Comment (shalousun) [inline src/main/java/com/ly/doc/builder/javadoc/JavadocDocBuilderTemplate.java]: Here, it is mandatory to override the previously configured Framework value. Users can generate various types of documentation using the basic configuration of smart-doc.

**Comment by shalousun** [comment]
It may be necessary to add support for exporting to Markdown, as some community users prefer to place Markdown files on various document management platforms. Consolidating the documentation into a single Markdown file would suffice.

**Comment by tuichenchuxin** [comment]
> It may be necessary to add support for exporting to Markdown, as some community users prefer to place Markdown files on various document management platforms. Consolidating the documentation into a single Markdown file would suffice.

@shalousun Thank you for your review, I will continue to modify it according to what you said.

**Comment by tuichenchuxin** [comment]
@shalousun I have modified it according to your suggestion. Please help me take another look.

File: src/main/resources/template/javadoc/Javadoc.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# ${htmlEscape(desc)}
+
+**Service:** ${name}
```
Comment (shalousun) [inline src/main/resources/template/javadoc/Javadoc.md]: The Service should be changed to Class; please modify it in other templates as well.

**Comment by shalousun** [comment]
@tuichenchuxin This PR doesn't have any major issues; you can rebase the commit message. After rebasing, it can be merged first. Later on, please also modify the smart-doc-maven-plugin.

Rebase can refer to this document. https://smart-doc-group.github.io/#/community/rebase-option

**Comment by tuichenchuxin** [comment]
> @tuichenchuxin This PR doesn't have any major issues; you can rebase the commit message. After rebasing, it can be merged first. Later on, please also modify the smart-doc-maven-plugin.
> 
> Rebase can refer to this document. https://smart-doc-group.github.io/#/community/rebase-option

@shalousun Thanks for the guidance, it has been modified.

**Review by shalousun — APPROVED** [review]

**Pull request merged**